pyiron_base/job/generic.py

```diff
@@ -6,9 +6,11 @@
 """
 
 from datetime import datetime
-import h5io
 import os
 import posixpath
+from typing import Union
+import h5io
```

Comment:
```suggestion
```
Doesn't seem to be used anymore.

Reply:
The h5io is used, typing not. I modified your suggestion to commit only that part.